src/vg/civcraft/mc/namelayer/command/commands/InvitePlayer.java

```diff
@@ -109,6 +109,7 @@ public boolean execute(CommandSender sender, String[] args) {
 		if(invitee.isOnline()){
 			//invitee is online make them a player
 			Player oInvitee = (Player) invitee;
+			Bukkit.getPluginManager().callEvent(new GroupInviteEvent(group, oInvitee, pType, p));
```

Comment:
You need to check if the event was cancelled after it was fired and to escape if it was.